### DISCUSSION TITLE
Mobile Responsiveness

### DISCUSSION THREAD

**Discussion by 99bits** [nhn/tui.calendar#1120] [General]
Hi, 

Thanks for the amazing work on this project.
I am researching various Event Calendars to use in my project that is for the web and on mobile.
I tested the calendar example for mobile responsiveness by resizing my browser.  The calendar resized but did not seem to be fully mobile compliant.  

Is there a Mobile mode for the Calendar?  

Thank you.

**Comment by adhrinae** [comment]
No, We don't support mobile devices at this moment.
